.pre-commit-config.yaml

```
@@ -3,11 +3,10 @@ repos:
   rev: 22.10.0
   hooks:
   - id: black
-- repo: https://github.com/asottile/reorder_python_imports
-  rev: v3.9.0
+- repo: https://github.com/pycqa/isort
+  rev:  5.11.4
```

```suggestion
  rev:  5.11.5
```
This is the version that solves the pre-commit issues for Python 3.7